modules/manage/partials/mountable-topics.adoc

```diff
@@ -7,7 +7,14 @@ For topics with Tiered Storage enabled, you can unmount a topic to safely detach
 include::shared:partial$enterprise-license.adoc[]
```

Comment:
@mattschumpert we do not have licensing content for the cloud docs like we do for self-managed. Should I remove/hide the licensing prereq for the Cloud version of this page?